src/frontend/src/crypto/ed25519.test.ts

```diff
@@ -66,9 +66,9 @@ const testVectorsSLIP10 = [
 test("derive Ed25519 via SLIP 0010", async () => {
   await Promise.all(
     testVectorsSLIP10.map(async (testVector: TestVector) => {
-      const seedBlob = fromHexString(testVector.seed);
-      const expectedPrivateKey = fromHexString(testVector.privateKey);
-      const expectedPublicKey = fromHexString(testVector.publicKey);
+      const seedBlob = fromHex(testVector.seed);
```

Comment:
Had a quick look to `fromHex` which seems a bit less efficient than `fromHexString`. Of course it's relative, it's just an additional loop but, out of curiosity any reason to change those?

Reply:
Yes, `fromHexString` is no longer available. And this is what the `agent-js` internal calls were changed to too, so I figured that this is the official replacement.

Reply:
Actually `fromHexString` is still available in `@dfinity/candid`. You used to include it using an incorrect path `@dfinity/identity/lib/cjs/buffer` which I guess already referenced the inherited implemented (just a guess). From having a look at the compiled code, it seems to confirm it so, long story short, if you just want to adapt the code without modifying the feature, I've got the feeling that you should stick to `fromHexString` but imported from `@dfinity/candid`.

That said, `fromHex` might work, just pointing out that those are two different implementations.

Reply:
Ok, thanks. I switched to the `candid` implementation now.